### PR TITLE
Remove ability to drop privileges from the top navigation bar

### PR DESCRIPTION
**Pull request by larskarlitski**
This is @stefwalter's commit from #9440 and various follow-up css fixes by @garrett.

### CODE REVIEW

File: pkg/shell/shell.less [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
+* (at your option) any later version.
+*
+* Cockpit is distributed in the hope that it will be useful, but
+* WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+* Lesser General Public License for more details.
+*
+* You should have received a copy of the GNU Lesser General Public License
+* along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+*/
```
Comment (stefwalter) [inline pkg/shell/shell.less]: Why was this code changed? Seems like churn.

File: pkg/shell/shell.less [hunk 2]
```diff
@@ -598,7 +557,7 @@ li.credential-lock > a:first-child {
 
 /* server overview dashboard */
 #server .info-table-ct button {
-  max-width: 300px;
+    max-width: 300px;
```
Comment (stefwalter) [inline pkg/shell/shell.less]: Again, indentation  fixes seem like needless churn, no?

File: pkg/shell/shell.less [hunk 3]
```diff
@@ -616,41 +575,41 @@ li.credential-lock > a:first-child {
         min-width: 220px;
     }
 
-  /* server overview dashboard */
-  #server .info-table-ct-container td button {
-      max-width: 100%;
-  }
+    /* server overview dashboard */
+    #server .info-table-ct-container td button {
+        max-width: 100%;
```
Comment (stefwalter) [inline pkg/shell/shell.less]: Ditto.

**Comment by garrett** [comment]
A lot of editors auto-detect indentation, and this file comes up as single space indentation, as evidenced by several people adding CSS at all sorts of indentation levels. Having all sorts of indentations messes with the clarity when reading the file. Fixing indentation prevents this problem.

**Comment by stefwalter** [comment]
> A lot of editors auto-detect indentation, and this file comes up as single space indentation, as evidenced by several people adding CSS at all sorts of indentation levels. Having all sorts of indentations messes with the clarity when reading the file. Fixing indentation prevents this problem.

Fixing indentation breaks branching, cherry-picking, backporting, code review and many other things we need as a team. This is especially the case when mixed in on another pull request.

This is why in Cockpit we have always only fixed indentation on only the lines that we change.

But @martinpitt is the team lead so I'll leave the call to him, whether this is okay from here on out.

Nevertheless, the indentation of the editor on the comment at the top is broken. This is valid indentation of a multiline comment:

```
/*
 * xxx
 */
```

This is not:

```
/*
* xxx
*/
```

**Comment by larskarlitski** [comment]
Fixed the misaligned comment at the top.

**Comment by garrett** [comment]
- Having a 1-space multiline comment like that still triggers 1-space indentation
- Caching is invalid as soon as we touch the file for any reason

But, yeah cherry-picking, backporting, code review are important. We could just drop the indentation changes here, I guess.

**Comment by martinpitt** [comment]
FTR, we do so few cherry-picks and backports (none at all in half a year, and maybe five patches in the year before) that adjusting for some space changes is not a problem at all. So please keep indentation changes in separate commits, otherwise I actually appreciate them. We recently started to enforce indentation in ES6 code, too.

**Comment by larskarlitski** [comment]
> This looks okay so far, but doesn't work consistently. When logging in with privs and clicking on "Unlocked", it goes away entirely. But when reloading the page afterwards or logging in without privs straight away it shows "Locked" instead. Please add another test case with locking in without privs to cover this. Thanks!

Hm, were you on a different branch? The point of this one is to remove the "Unlock" button completely. It should now show a non-clickable "Privileged" label in the header if you logged in with the "use my credentials to escalate"-box checked.

**Pull request merged**